mlflow/azureml/__init__.py

```diff
@@ -31,6 +31,16 @@ def build_image(model_path, workspace, run_id=None, image_name=None, model_name=
     The resulting image can be deployed as a web service to Azure Container Instances (ACI) or
     Azure Kubernetes Service (AKS).
 
+    The resulting Azure ML ContainerImage will contain a webserver that accepts the following
+    data formats as input:
+
+    - JSON-serialized Pandas DataFrames in the `split` orientation. For example,
```

Comment:
Maybe this can also link to the central place of documentation, and can say explicitly that this always assumes the split format.